lib/Monocle_Alternate.php

```diff
@@ -64,6 +64,53 @@ public function get_active($eids, $miniv, $minlevel, $exminiv, $swLat, $swLng, $
         return $this->query_active($select, $conds, $params);
     }
 
+    public function get_active_by_id($ids, $miniv, $minlevel, $exminiv, $swLat, $swLng, $neLat, $neLng)
+    {
+        global $db;
+        $conds = array();
+        $params = array();
+
+        $select = "pokemon_id, expire_timestamp AS disappear_time, encounter_id, lat AS latitude, lon AS longitude, gender, form";
```

Comment:
Not sure where you copied this from but this line has `weight` on develop branch